<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(25,35,45);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>129</y>
      <width>1181</width>
      <height>721</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout"/>
   </widget>
   <widget class="QLabel" name="imgLabel">
    <property name="geometry">
     <rect>
      <x>1006</x>
      <y>0</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{border-image:url(&quot;C:/Users/javen/Desktop/高清Logo/白色条形.png&quot;);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="netCardLabel">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>0</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
font-family:宋体;
/*字体大小为20点*/
font-size:10pt;</string>
    </property>
    <property name="text">
     <string>当前选择网卡：                    </string>
    </property>
   </widget>
   <widget class="QPushButton" name="BADButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>80</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:#ed1941;
    /*边框圆角半径为8像素*/ 
    border:3px solid black;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:#d71345;
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:#d71345;
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>恶意包</string>
    </property>
   </widget>
   <widget class="QPushButton" name="UDPButton">
    <property name="geometry">
     <rect>
      <x>99</x>
      <y>80</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(80,95,105);
    /*边框圆角半径为8像素*/ 
    border:3px solid rgb(0,0,0);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>UDP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OTHERButton">
    <property name="geometry">
     <rect>
      <x>373</x>
      <y>80</y>
      <width>82</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(80,95,105);
    /*边框圆角半径为8像素*/ 
    border:3px solid rgb(0,0,0);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>其他</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TCPButton">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>80</y>
      <width>82</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(80,95,105);
    /*边框圆角半径为8像素*/ 
    border:3px solid rgb(0,0,0);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>TCP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ARPButton">
    <property name="geometry">
     <rect>
      <x>282</x>
      <y>80</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(80,95,105);
    /*边框圆角半径为8像素*/ 
    border:3px solid rgb(0,0,0);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>ARP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ICMPButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>82</width>
      <height>29</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(80,95,105);
    /*边框圆角半径为8像素*/ 
    border:3px solid rgb(0,0,0);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(50,59,65);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>ICMP</string>
    </property>
   </widget>
   <widget class="QPushButton" name="chooseNetCard">
    <property name="geometry">
     <rect>
      <x>-2</x>
      <y>0</y>
      <width>102</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:#f3704b;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:#f15a22;
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:#f15a22;
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fileButton">
    <property name="geometry">
     <rect>
      <x>305</x>
      <y>0</y>
      <width>102</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(142,133,112);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(122,113,92);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(122,113,92);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>203</x>
      <y>0</y>
      <width>102</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:rgb(80,95,105);
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:rgb(50,65,75);
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:rgb(50,65,75);
    /*左内边距为3像素，让按下时字向右移动3像素*/  
    padding-left:3px;
    /*上内边距为3像素，让按下时字向下移动3像素*/  
    padding-top:3px;
}
</string>
    </property>
    <property name="text">
     <string>清空记录</string>
    </property>
   </widget>
   <widget class="QPushButton" name="stopButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/*按钮普通态*/
QPushButton
{
    /*字体为微软雅黑*/
    font-family:宋体;
    /*字体大小为20点*/
    font-size:14pt;
    /*字体颜色为白色*/    
    color:white;
    /*背景颜色*/  
    background-color:#101010;
}

/*按钮停留态*/
QPushButton:hover
{
    /*背景颜色*/  
    background-color:#130c0e;
}

/*按钮按下态*/
QPushButton:pressed
{
    /*背景颜色*/  
    background-color:#130c0e;
    /*左内边距为3像素，让按下时字向右移动3像素 
    padding-left:3px;
    padding-right:3px;*/ 
    /*上内边距为3像素，让按下时字向下移动3像素  
    padding-top:3px;*/
}
</string>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>80</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;</string>
    </property>
    <property name="text">
     <string>注：双击可查看详细信息</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>瞬时恶性包率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="rlabel">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>80</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;</string>
    </property>
    <property name="text">
     <string>总恶性包率：</string>
    </property>
   </widget>
   <widget class="QLabel" name="rlabel_2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
